src/planscape/impacts/serializers.py

```diff
@@ -31,23 +36,69 @@ class Meta:
 
 
 class TreatmentPlanListSerializer(serializers.ModelSerializer):
-    pass
+    def get_creator_name(self, instance):
+        return instance.user.get_full_name()
+
+    class Meta:
+        model = TreatmentPlan
+        fields = (
+            "uuid",
+            "created_at",
+            "created_by",
+            "creator_name",
+            "status",
+            "name",
+        )
 
 
 class TreatmentPrescriptionSerializer(serializers.ModelSerializer):
     class Meta:
         model = TreatmentPrescription
+        fields = (
+            "uuid",
+            "created_at",
+            "created_by",
+            "updated_at",
+            "updated_by",
+            "treatment_plan",
+            "project_area",
+            "stand",
+            "action",
+            "geometry",
+        )
+
+
+class UpsertTreamentPrescriptionSerializer(serializers.Serializer):
```

Comment:
nit: Some of our serializers have names that start with the model name, some with the verb/action. I don't have strong opinions on which convention we should adopt, but I suppose we ought to pick one.